packages/manager/modules/telecom-spare/src/modems/order/modems-order.html

```diff
@@ -0,0 +1,236 @@
+                            <div>
+                                <h3 data-translate="modems_order_select_title"></h3>
+                                <select class="form-control" data-ng-model="$ctrl.order.brand">
+                                    <option data-ng-repeat="brand in $ctrl.brands" value="{{brand.id}}">{{brand.name}}</option>
+                                </select>
+                                <div>
+                                    <label data-translate="modems_order_select_quantity"></label>
+                                    <oui-numeric data-model="$ctrl.quantity" data-max="50"></oui-numeric>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <button type="button"
+                            class="oui-button oui-button_primary oui-button_icon-right float-right mt-2"
+                            data-ng-disabled="($ctrl.quantity === 0) || !$ctrl.order.brand || !$ctrl.order.contact || ($ctrl.order.shipping.mode === 'mondialRelay' && !$ctrl.order.shipping.relay)"
+                            data-ng-click="$ctrl.changeStep('summary')">
+                        <span data-translate="modems_order_order_next"></span>
+                        <i class="oui-icon oui-icon_circle oui-icon-chevron-right" aria-hidden="true"></i>
+                    </button>
+                </section>
+
+                <!-- SUMMARY -->
+                <section class="widget-presentation"
+                         data-ng-if="$ctrl.orderStep === 'summary'">
+                    <h2 class="widget-presentation-title" data-translate="modems_order_summary_title"></h2>
+
+                    <div class="text-center" data-ng-if="$ctrl.isStepLoading">
+                        <oui-spinner></oui-spinner>
+                    </div>
+
+                    <div data-ng-if="!$ctrl.isStepLoading">
+
+                        <!-- PRICE SUMMARY TABLE -->
+                        <h3 data-translate="modems_order_summary"></h3>
+                        <div class="table-responsive">
+                            <table class="table"
+                                    data-ng-class="{
+                                        'table-hover table-striped': $ctrl.order.summary.details.length > 1
+                                    }">
+                                <thead>
+                                    <tr>
+                                        <th scope="col"
+                                            data-translate="modems_order_table_description">
+                                        </th>
+                                        <th scope="col"
+                                            class="text-sm-right"
+                                            data-translate="modems_order_table_unit_price"
+                                            style="width:25%;">
```

Comment:
```suggestion
>
```
If this is really needed should be either a bootstrap utility or in a stylesheet